.changeset/green-stingrays-live.md

```diff
@@ -0,0 +1,9 @@
+---
+'@kadena/wallet-adapter-snap': major
```

Comment:
I've been avoiding `major` to keep versions below 1.0. We did stop using `beta` tag but I'd prefer our packages to be seen as an early project still adapting to feedback where needed. I'll leave this one up to you though, it's your package. If you feel like `1.0.0` is appropriate then that's all good.

Reply:
Thanks for the context, let's keep it minor then. I think it makes sense to use the same versioning as the other adapter packages